tests/Hooks/Migration/Unit/ConfiguresMigrationsTest.php

```diff
@@ -29,21 +29,45 @@ class ConfiguresMigrationsTest extends ConfigureHookTestCase
 
     protected $eventClass = ConfigureMigrations::class;
 
+    protected function afterSetUp()
+    {
+        resolve('migrator')->path(__DIR__);
+        parent::afterSetUp();
+    }
+
```

Comment:
Should this really be in the `afterSetup` and not simply in the tests that need this?

Reply:
It needs to be set BEFORE the hook constructor gets called, which is done in the parent::afterSetUp()